fsdriver/driver_basic_darwin.go

```diff
@@ -0,0 +1,11 @@
+package fsdriver
+
```

Comment:
hmm... I just moved the file from driver_darwin.go to fsdriver/driver_basic_darwin.go and changed the package name. Seems like driver_darwin.go can just be deleted from the master branch.